### DISCUSSION TITLE
cannot detect client ip

### DISCUSSION THREAD

**Discussion by orangmiliter** [AdguardTeam/AdGuardHome#3211] [Q&A]
cannot detect client ip of mi router 4a
version : v0.106.3

**Comment by ameshkov** [comment]
Most likely your router proxies all queries so AGH can only see your router's IP address. I am not sure if this can be solved, let's see if anyone else here has any idea.

**Comment by markhicken** [comment]
I was having this problem on a Ubiquiti Dream Machine even though the WAN's DNS was set to my local AGH server. This is how I fixed it...
1. Settings > Networks > MY_NETWORK (edit)
2. Set `DHCP Name Server` to `manual` and add the IP address of AGH

![Screen Shot 2021-06-10 at 10 30 14 AM](https://user-images.githubusercontent.com/849930/121562897-24f32c00-c9d7-11eb-81c6-d10e2f5f98fe.png)

**Reply by orangmiliter** [reply]
I am like that too
